art/art/urls.py

```diff
@@ -13,9 +13,20 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.contrib import admin
+
+from django.contrib import admin 
+from django.conf.urls import url, include
 from django.urls import path
+from rest_framework.documentation import include_docs_urls
+
+API_TITLE = 'Art.API'
+API_DESCRIPTION = 'This a documentation of all the API endpoints being used in this project, what they do and how to use them.'
+
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
```

Comment:
There's a duplicate that references `admin.site.urls` that is in the main `urls.py` in the art project folder. Please have that removed.

Reply:
done